module.css

```diff
@@ -1,3 +1,19 @@
 .template {
-display: none;
+  display: none;
+}
+.nav-items {
```

Comment:
Your CSS currently isn't working because your list items that you're intending on styling have a class of 'nav-item' not 'nav-items'.